addons/l10n_ie/data/template/account.account-ie.csv

```diff
@@ -1,104 +1,738 @@
+"l10n_ie_account_3632","3632","Inventories of buildings for resale in foreign countries","asset_current","False"
+"l10n_ie_account_37","37","Down payments on account on inventories","asset_current","False"
+"l10n_ie_account_4011","4011","Customers due within one year","asset_receivable","True"
+"l10n_ie_account_40111","40111","Customers (PoS)","asset_receivable","True"
+"l10n_ie_account_4012","4012","Customers - Receivable bills of exchange","asset_current","True"
+"l10n_ie_account_4013","4013","Doubtful or disputed customers","asset_current","True"
+"l10n_ie_account_4014","4014","Customers - Unbilled sales","asset_current","True"
+"l10n_ie_account_4015","4015","Customers with a credit balance","liability_current","True"
+"l10n_ie_account_4019","4019","Value adjustments","asset_current","False"
+"l10n_ie_account_4021","4021","Customers due after more than one year","asset_receivable","True"
+"l10n_ie_account_4025","4025","Customers with creditor balance","asset_current","True"
+"l10n_ie_account_4029","4029","Value adjustments","asset_current","False"
+"l10n_ie_account_41111","41111","Trade receivables","asset_current","False"
+"l10n_ie_account_41112","41112","Loans and advances","asset_current","False"
+"l10n_ie_account_41118","41118","Other receivables","asset_current","False"
+"l10n_ie_account_41119","41119","Value adjustments","asset_current","False"
+"l10n_ie_account_41121","41121","Trade receivables","asset_current","False"
+"l10n_ie_account_41122","41122","Loans and advances","asset_current","False"
+"l10n_ie_account_41128","41128","Other receivables","asset_current","False"
+"l10n_ie_account_41129","41129","Value adjustments","asset_current","False"
+"l10n_ie_account_41211","41211","Trade receivables","asset_current","False"
+"l10n_ie_account_41212","41212","Loans and advances","asset_current","False"
+"l10n_ie_account_41218","41218","Other receivables","asset_current","False"
+"l10n_ie_account_41219","41219","Value adjustments","asset_current","False"
+"l10n_ie_account_41221","41221","Trade receivables","asset_current","False"
+"l10n_ie_account_41222","41222","Loans and advances","asset_current","False"
+"l10n_ie_account_41228","41228","Other receivables","asset_current","False"
+"l10n_ie_account_41229","41229","Value adjustments","asset_current","False"
+"l10n_ie_account_42111","42111","Advances and down payments","asset_current","False"
+"l10n_ie_account_42119","42119","Value adjustments","asset_current","False"
+"l10n_ie_account_4212","4212","Amounts owed by partners and shareholders (others than from affiliated undertakings)","asset_current","False"
+"l10n_ie_account_42131","42131","Investment subsidies","asset_current","False"
+"l10n_ie_account_42132","42132","Operating subsidies","asset_current","False"
+"l10n_ie_account_42138","42138","Other subsidies","asset_current","False"
+"l10n_ie_account_42141","42141","Corporate income tax","asset_current","False"
```

Comment:
I'm not sure I understand why corporate income tax is an asset. Surely it should be a liability, no? Unless it's an expected tax refund owed by the government?

Same for the other taxes - shouldn't they be liabilities?